lib/powerpoint.js

```diff
@@ -0,0 +1,79 @@
+var app = require('app')
+var BrowserWindow = require('browser-window')
+var officegen = require('officegen')
+var pptx = officegen('pptx')
+var fs = require('fs')
+var ipc = require('ipc')
+
+module.exports = () => {
+    var out = fs.createWriteStream(app.windows[0].getTitle() + '.pptx')
+    var deckLength = 0
+    var win = app.windows[0]
+    var pdfWin = new BrowserWindow({
+        toolbar: false,
+        center: true,
+        width: 987,
+        height: 680,
+        show: false
+    })
+
+    pdfWin.loadUrl('http://localhost:2000/')
+    setTimeout(function () {
+        pdfWin.webContents.executeJavaScript(`
+            var ipc = require('ipc')
+            ipc.on('asynchronous-message', (event, arg) => {
+                ipc.send('asynchronous-reply', BESPOKE_PDF.slides.length)
+            })
+        `)
+        pdfWin.webContents.send('asynchronous-message', 'ping')
+    }, 1000)
+
+    var index = 1
+    function createPowerpoint () {
+        setTimeout(function () {
+            progress(index, deckLength)
+            var url = 'http://localhost:2000/#' + index
+            pdfWin.loadUrl(url)
+            setTimeout(function () {
+                pdfWin.capturePage(image => {
+                    slide = pptx.makeNewSlide()
```

Comment:
is this an accidental global? if you don't declare with var that's going to end up on `global`